### ISSUE TITLE
Create Connections Screen Improvement Suggestion

### ISSUE THREAD

**Issue by lhein** [syndesisio/syndesis-ui#1072]
When opening that screen for the first time with no connection existing you are seeing this:

![bildschirmfoto von 2017-10-11 11-29-48](https://user-images.githubusercontent.com/435853/31432805-d23abad8-ae77-11e7-9985-3d0a35d938a2.png)

There is a nice and easy to find "Create Connection" button right in the center of the screen...you can't miss it. 

However once you created your first connection then this screen comes up....

![bildschirmfoto von 2017-10-11 11-30-14](https://user-images.githubusercontent.com/435853/31432852-f5dd29a8-ae77-11e7-8fb4-b10bdd57cf9d.png)

Now where is the nice big create button? It is gone? No...there is this tiny little button on the top right of the screen you will most likely miss at the very beginning...

I think you should not remove the big button. Just append it or even better let it be the first entry always so no user will miss it. That done the upper right button can be replaced by something else.

Just my thoughts...

**Comment by gashcrumb** [comment]
Yeah, it'd be pretty easy to add the same 'create connection' tile that shows up in the integration editor at the end of the list...

**Comment by rhuss** [comment]
@dongniwang what is your opinion on this issue ?

**Comment by dongniwang** [comment]
Thanks for the feedback @lhein !

First of all, the call to action button placement is following PatternFly [Toolbar](http://www.patternfly.org/pattern-library/forms-and-controls/toolbar/) pattern. Good to know that the discoverability of the button is not very good based on your experience. 

Secondly, there's a concern with placing the create button in the center of the screen when users have many connection cards on this page. In that case, the create button at the top right corner would offer easier access for users. 

> Yeah, it'd be pretty easy to add the same 'create connection' tile that shows up in the integration editor at the end of the list...

Yes. This would help. But I'm wondering we should also drive users' attention to the toolbar create button. What do you think? @sjcox-rh @amysueg

**Comment by sjcox-rh** [comment]
Yes, thanks for the feedback @lhein! And yeah I agree @dongniwang that we should still have the Create Connection button in the toolbar. Mainly to stay consistent throughout the app because in the integration list page, the Create Integration button is also found in the toolbar.

Also, agree the create connection card would help, but if it's in the end of the list, we risk the fact that it will eventually fall off the screen as the user adds more connections.

**Comment by amysueg** [comment]
Yes, thank you @lhein for the feedback.

Agree with both @dongniwang and @sjcox-rh ton adding a create connect card at the end, in addition to keeping the button on the toolbar. I don't think dropping off the screen after a number of connections added would be a problem, since by that point the user (perhaps) will know how to add connections.

**Comment by lhein** [comment]
>  Just append it **or even better let it be the first entry always so no user will miss it**. That done 

I am quoting myself just to make sure you got my point.
